docs/questions/sharing/visualizations/gauge.md

```diff
@@ -4,10 +4,57 @@ title: Gauge chart
 
 # Gauge chart
 
-Ah, **gauges**: you either love 'em or you hate 'em. …Or you feel "meh" about them, I guess. Whatever the case, gauges allow you to show a single number in the context of a set of colored ranges that you can specify. By default, when you choose the Gauge visualization, Metabase will create red, yellow, and green ranges for you.
+Ah, **gauges**: you either love 'em or you hate 'em. …Or you feel "meh" about them, I guess. Whatever the case, gauges allow you to show a single number in the context of a set of colored ranges that you can specify.
 
 ![Gauge](../../images/gauge.png)
 
-Open up the visualization settings to define your own ranges, choose colors for them, and optionally add labels to some or all of your ranges:
+## When to use a gauge chart
 
-![Gauge settings](../../images/gauge-settings.png)
\ No newline at end of file
+A gauge chart is useful when you want to show progress or status across different categories.
+They're helpful for performance metrics, risk assessments, or any metric that has distinct thresholds.
+
+## How to create a gauge chart
+
+To build a gauge chart, you'll need:
+
+- A query that returns a single number:
+
+  | Sum of Quantity |
+  | --------------- |
+  | 4910            |
+
+  You can also use a query that returns several metrics in a single row:
+
+  | Sum of Quantity | Average Quantity | Max Quantity |
+  | --------------- | ---------------- | ------------ |
+  | 4910            | 17.32            | 173          |
+
+  In this case only the metric in the first column will be used for the gauge chart. To change which number is used for the gauge, rearrange the metrics (for example, by dragging them in the Summarize block in the query builder).
```

Comment:
the "for example" part is because I've seen people reorder columns in the table viz first and then build a chart. This won't work - they need to be reordered in the query itself